application/logging_default_config.yaml

```diff
@@ -0,0 +1,34 @@
+version: 1
+disable_existing_loggers: False
+
+formatters:
+    default_formatter:
+        format: '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+        datefmt: '%Y-%m-%d %H:%M:%S'
+
+handlers:
+    console:
+        class: logging.StreamHandler
+        level: DEBUG
+        formatter: default_formatter
+        stream: ext://sys.stdout
+
+loggers:
+    development:
+        level: DEBUG
+        handlers: [console]
+        propagate: no
+
+    staging:
+        level: INFO
+        handlers: [console]
+        propagate: no
+
+    production:
+        level: WARNING
+        handlers: [console]
+        propagate: no
```

Comment:
That's some good thinking but none of the loggers are getting used if we use `__file__` to `getLogger()`, right?

```suggestion
```

Reply:
In the root logger configuration, I specified it to use the console handler, so that one will be used. I tested in my own local environment and the output message looks correct.